docs/pages/blog/turbo-1-7-0.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Turborepo 1.7
+date: 2023/01/11
+description: Turborepo 1.7 focuses on improving developer experience by bringing more clarity to your tasks.
+tag: "web development"
+ogImage: /images/blog/turbo-1-7-0/twitter-card.png
+---
+
+import { Tabs, Tab } from '../../components/Tabs'
+
+# Turborepo 1.7
+
+import { Authors } from '../../components/Authors'
+import Badge from '../../components/Badge'
+import Date from "../../components/blog/Date";
+
+<Date>
+  Wednesday, January 11th, 2023
+</Date>
+
+<Authors authors={[
+  'gregsoltis',
+  'nathanhammond',
+  'tomknickman',
+  'anthonyshew',
+  'jaredpalmer',
+  'mehulkar',
+  'chrisolszewski',
+  'nicholasyang'
```

Comment:
just curious... are we just copy pasting everyone onto every post? seems like it would be nicer if we filtered this list by whoever's actually writing (and acknowledge code contributions in the post itself if we want to do that)

Reply:
The whole team is typically involved with the release blog posts in some aspect (from writing to reviewing to publishing) - so it's simpler in my opinion to list the core team as the authors. Plus we're all hashing out all the content day to day as a team anyway. We're a team and we're all involved here.  

For the more specific one offs (like https://turbo.build/blog/turbopack-benchmarks) I think those make sense to list _writing_ authors only. 

I could be convinced otherwise as well, but I don't really feel the need to be pedantic about the bar for entry to a release post.

Reply:
Sounds good, just gut checking :). 

> We're a team and we're all involved here.

💯 💟